lsqecc/pauli_rotations/circuit.py

```diff
@@ -1,7 +1,7 @@
 import copy
 import pyzx as zx
 from fractions import Fraction
-from typing import List, cast
+from typing import List, cast, Type
```

Comment:
```suggestion
from typing import List, cast
```
Seems like `Type` was not used here?

Reply:
Mmmm I wonder why black diddn't remove it